File: presentation/src/main/java/com/bunbeauty/presentation/item/CafeItem.kt
package com.bunbeauty.presentation.item

import com.bunbeauty.domain.model.BaseItem

data class CafeItem(
    override var uuid: String,
    val address: String,
    val workingHours: String,
    val isOpenMessage: String,
    val isOpenColor: Int,
) : BaseItem()

File: PapaKarloSwift/PapaKarloSwift/feature/cafe/cafe_list/CafeItem.swift
//
//  CafeItem.swift
//  PapaKarloSwift
//
//  Created by Марк Шавловский on 09.03.2022.
//
import SwiftUI
import Foundation

struct CafeItem:Identifiable {
    let id:UUID
    let address:String
    let workingHours:String
    let isOpenMessage:String
    let isOpenColor:Color
}

